### PR TITLE
-Fixed typo in docs

### PR DESCRIPTION
**Pull request by lorenzo-gomez-windhover**

### CODE REVIEW

File: core/framework/doc/index.rst [hunk 1]
```diff
@@ -82,4 +82,4 @@ With the use of adapters and the selection service, each application simply need
 
 Register zero or more
 `AdapterFactories <https://github.com/ControlSystemStudio/phoebus/blob/master/app/logbook/ui/src/main/java/org/phoebus/logbook/ui/adapters/AppSelectionAdapterFactory.java>`_
-which provide the mechanism to adapt an AppSelection to a simple LogEntry or an Olog LogEnrty
\ No newline at end of file
+which provide the mechanism to adapt an AppSelection to a simple LogEntry or an Olog LogEnrty
```
Comment (kasemir) [inline core/framework/doc/index.rst:85]: While at it, can you fix `LogEnrty` -> `LogEntry`?

Comment (lorenzo-gomez-windhover) [reply core/framework/doc/index.rst:85]: It looks like you guys have `docs/core/` and a bunch of other documentation files under the git ignore:
https://github.com/ControlSystemStudio/phoebus/blob/master/docs/source/.gitignore

Are the contents of these files auto-generated by something?

**Comment by shroffk** [comment]
The appliation.rst and a few other files are programmatically generated and hence added to gitignore.
We wanted the documentation to live within each application module (better for maintenance)... when the documentation is built, all the docs from all the modules are pulled together and the appropriate links are created.

**Comment by lorenzo-gomez-windhover** [comment]
I see what you mean. Good to know.
Thanks!

**Comment by shroffk** [comment]
Thank you for fixing all these typos :)

**Pull request merged**